inst/json/metadata_0.1.0.json

```diff
@@ -75,9 +75,11 @@
       { "id": "business_closures", "label": "Business closures" }
     ],
     "time_series": [
-      { "id": "cases", "label": "Cases", "description": "Number of confirmed cases" },
-      { "id": "vacc", "label": "Vaccinated", "description": "Cumulative vaccinations" },
-      { "id": "hosp_occ", "label": "Hospital Occupancy", "description": "Number of hospitalised cases" }
+      { "id": "infect_asymp", "label": "Asymptomatic Infections", "description": "Infections without symptoms" },
+      { "id": "infect_symp", "label": "Symptomatic Infections", "description": "Infections with symptoms" },
+      { "id": "hospitalised", "label": "Hospital Occupancy", "description": "Number of hospitalised cases" },
```

Comment:
In our results, we get stuff like at a given timestep, several million 'hospitalised', which is far more than the hospital surge capacity (a few tens of thousands). In these cases, calling it hospital 'occupancy', or indeed number of 'hospitalised' cases, isn't quite correct, but I don't have a better suggestion. This naming issue derives from the model itself, of course, which has a hospitalisation compartment that (I guess) isn't capped by the actual hospital capacity - I am assuming 'hospitalisation' basically refers to a severity of illness.

Reply:
@pratikunterwegs Is my assumption correct, that the compartment called 'hospitalised' is more about the number of people with a serious illness, than being about the number of people located in a hospital?

Reply:
We assume it's serious illness requiring hospitalisation and theoretically, located in a hospital. When the number of hospitalised exceeds the spare capacity, the mortality rate is increased reflecting the pressure on the medical system.